cluster/saltbase/salt/rescheduler/rescheduler.manifest

```diff
@@ -28,8 +28,8 @@ spec:
     # TODO: split this out into args when we no longer need to pipe stdout to a file #6428
     - sh
     - -c
-    - '/rescheduler --running-in-cluster=false 1>>/var/log/rescheduler.log 2>&1'
+    - '/rescheduler --listen-address=127.0.0.1:9235 --running-in-cluster=false --log_dir=/var/log/rescheduler --logtostderr=false'
```

Comment:
Just a question here, why need to add `listen-address` here? The default value is already localhost:9235 according to https://github.com/kubernetes/contrib/blob/master/rescheduler/rescheduler.go#L84